review/fileToReview.js

```diff
@@ -12,62 +17,42 @@ var domain = "bank.local.fr"
  * @return {Object} All transactions available on the page
  */
 async function fetchTransactions(fromDate, authorization, jws = null, id, page, previousTransactions) {
-	console.log(`--- Fetch Trasactions page n°${page} ---`);
-	try {
-    var headers = {"Authorisation":  authorization }
+  console.log(`--- Fetch Trasactions page n°${page} ---`);
 
-    if (jws) {
-      headers = {
-        "Authorisation": authorization,
-        "jws": jws,
-        "Content-type": "application/json",
-        "Accept": "application/json"
-      }
-    } else {
-      headers = {
-        "Authorisation": authorization,
-        "Content-type": "application/json",
-        "Accept": "application/json",
-      }
-    }
+  const headers = jws ? {
+    ...config.defaultHeaders,
+    "jws": jws,
+    "Authorisation": authorization
+  } : {...config.defaultHeaders, "Authorisation": authorization};
+
+  const {code, response} = await doRequest('GET', `${config.domain}/accounts/${id}/transactions?page=${page}`, headers);
 
-	  var {code, response } = await doRequest('GET',
```

Comment:
The url is not clear at all, and can be factorized as : `${config.domain}/accounts/${id}/transactions?page=${page}`